app/src/main/java/com/kickstarter/viewmodels/ThreadViewModel.kt

```diff
@@ -407,7 +406,10 @@ interface ThreadViewModel {
             /** reversed replies **/
             apolloPaginate
                 .paginatedData()
-                ?.map { it.reversed() }
+                ?.map {
+                    it.toMutableList().reverse()
```

Comment:
fyi @mtgriego could be related with the issues you found when migrating Discovery to RXJava2

Reply:
ill look into it, thanks!